lib/definitions/time.js

```diff
@@ -1,4 +1,5 @@
-var time
+var time;
+var daysInYear = 365.25;
```

Comment:
This looks like a breaking change that's unrelated to the rest of this pull request. Can you make the date changes a separate pull request so that we can release it as its own major version? Otherwise, people will have to upgrade to a new major version to get the exclude feature.